cirq/google/arg_func_langs_test.py

```diff
@@ -126,12 +126,12 @@ def test_serialize_sympy_constants():
                                        including_default_value_fields=True,
                                        preserving_proto_field_name=True,
                                        use_integers_for_enums=True)
+    assert len(packed) == 1
+    assert len(packed['arg_value']) == 1
     # protobuf 3.12+ truncates floats to 4 bytes
-    assert packed == {
-        'arg_value': {
-            'float_value': float(str(np.float32(sympy.pi)))
-        }
-    }
+    assert np.isclose(packed['arg_value']['float_value'],
+                      np.float32(sympy.pi), atol=1e-8)
```

Comment:
Is 1e-8 too tight?  Shouldn't it be 1e-6?

Reply:
Updated to 1e-7